mountpoint-s3/scripts/fs_latency_bench.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+if [[ -z "${S3_BUCKET_NAME}" ]]; then
+  echo "Set S3_BUCKET_NAME to run this benchmark"
+  exit 1
+fi
+
+if [[ -z "${S3_BUCKET_TEST_PREFIX}" ]]; then
+  echo "Set S3_BUCKET_TEST_PREFIX to run this benchmark"
+  exit 1
+fi
+
+if [[ -z "${S3_BUCKET_BENCH_FILE}" ]]; then
+  echo "Set S3_BUCKET_BENCH_FILE to run this benchmark"
+  exit 1
+fi
+
+if [[ -z "${S3_BUCKET_SMALL_BENCH_FILE}" ]]; then
+  echo "Set S3_BUCKET_SMALL_BENCH_FILE to run this benchmark"
+  exit 1
+fi
+
+base_dir=$(dirname "$0")
+project_dir="${base_dir}/../.."
+cd ${project_dir}
+
+results_dir=results
+jobs_dir=mountpoint-s3/scripts/fio
+target_gbps=100
+thread_count=4
+
+rm -rf ${results_dir}
+mkdir -p ${results_dir}
+
+# start readdir benchmarking
+dir_size=100
+while [ $dir_size -le 100000 ]
+do
+    sum=0
+    job_name="readdir_${dir_size}"
+    mount_dir=$(mktemp -d /tmp/mount-s3-XXXXXXXXXXXX)
+    target_dir="${mount_dir}/bench_dir_${dir_size}"
```

Comment:
we should update the benchmarking docs to describe how this directory structure is created

Reply:
Added a readdir section in the doc to explain this.